### PR TITLE
e2e(fixed rust-wasm sample and covered checks for it)

### PR DESCRIPTION
**Pull request by edchai**
Screenshot 2023-01-12 at 17 24 46

### CODE REVIEW

File: cypress/common/selectors.ts [hunk 1]
```diff
@@ -78,6 +78,7 @@ export const selectors = {
     appExposesCloseButton: '[data-e2e="exposesAppNamesClose"]',
```
Comment (PeaceDate) [inline cypress/common/selectors.ts:77]: Add please comma on 67th line

File: rust-wasm/e2e/methods/methods.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+    public checkGameBoardAppearsByClick(buttonName: string): void {
+        this.checkElementVisibility(selectors.rustWasmGameBoard, false)
+        this.clickElementWithText({
+            selector: baseSelectors.button,
+            text: buttonName
+        })
+        this.checkElementVisibility(selectors.rustWasmGameBoard)
+    }
+}
```
Comment (PeaceDate) [inline rust-wasm/e2e/methods/methods.ts]: Add BL

File: cypress/fixtures/constants.ts [hunk 3]
```diff
@@ -620,7 +628,13 @@ export class Constants {
...
             cloudSymbol: 'cloud'
         },
-        displayNone: 'display: none;'
+        displayNone: 'display: none;',
+        rustWasmAppButtonsNames: [
+            'Play ▶️',
+            'Tick 🔂',
+            'Reset ♻️',
+            'Stop 🛑'
+        ],
```
Comment (PeaceDate) [inline cypress/fixtures/constants.ts:637]: What do you think about it:
```
        rustWasmAppButtonsNames: {
            play: 'Play ▶️',
            tick: 'Tick 🔂',
            reset: 'Reset ♻️',
            stop: 'Stop 🛑'
        },
```

Comment (PeaceDate) [reply cypress/fixtures/constants.ts:637]: And then use it like that in test: 
```
Constants.commonText.rustWasmAppButtonsNames.play
Constants.commonText.rustWasmAppButtonsNames.tick
Constants.commonText.rustWasmAppButtonsNames.reset
Constants.commonText.rustWasmAppButtonsNames.stop
```

File: rust-wasm/e2e/methods/methods.ts (hunk 2 above)
Comment (edchai) [reply rust-wasm/e2e/methods/methods.ts]: fixed

File: cypress/common/selectors.ts (hunk 1 above)
Comment (edchai) [reply cypress/common/selectors.ts:77]: fixed

**Review by PeaceDate — APPROVED** [review]

**Pull request merged**